src/_scss/layouts/search/header/_downloadButton.scss

```diff
@@ -1,34 +1,111 @@
 .download-wrap {
     position: relative;
 
+    .download-hover {
+        position: absolute;
+        top: rem(-18);
+        left: rem(-290);
+        z-index: $z-modal - 1;
+
+        width: rem(270);
+        height: rem(90);
+
+        .download-hover-interior {
+            position: relative;
+        }
+
+        .hover-content {
+            @include display(flex);
+            @include flex-direction(row);
+            @include justify-content(center);
+            @include align-items(center);
+            position: relative;
+
+            background-color: $color-gold-lightest;
+            border: 1px solid $color-gold-light;
+            box-shadow: $box-shadow;
+
+            padding: rem(10);
+
+            .icon {
+                @include flex(0 0 auto);
+                width: rem(25);
+                height: rem(25);
+                margin-right: rem(10);
+                svg {
+                    fill: $color-gold;
+                    width: rem(25);
+                    height: rem(25);
+                }
+            }
+
+            .message {
+                @include flex(1 1 auto);
+                text-align: left;
+                color: $color-base;
+                font-size: $smallest-font-size;
+            }
+        }
+
+        // inherit from the standard tooltip arrow style
+        $color-tooltip-border: $color-gold-light;
+        @import "components/visualizations/tooltip/_arrow";
+        .tooltip-pointer {
+            // override the coloring
+            &:after {
+                background: $color-gold-lightest;
+            }
+
+            &.right {
+                top: rem(32);
+                right: rem(9);
+            }
+        }
+    }
+
     .download-button {
         @include button-unstyled;
-        color: $color-white;
-        text-transform: uppercase;
-        font-size: 1em;
-        font-weight: $font-semibold;
-        letter-spacing: 0.075em;
         padding: rem(10) rem(20);
         border: 1px solid $color-white;
         cursor: pointer;
 
-        &:hover, &:active {
+        @include display(flex);
+        @include justify-content(center);
+        @include align-items(center);
+        position: relative;
+        width: rem(155);
+
+        @include transition(opacity 0.2s ease-in-out);
+
+        .icon {
+            @include flex(0 0 auto);
+            width: rem(20);
+            height: rem(20);
+            margin-right: rem(10);
+            svg {
+                fill: $color-white;
+                width: rem(20);
+                height: rem(20);
+            }
+        }
+
+        .label {
+            @include flex(1 1 auto);
             color: $color-white;
-            border: 1px solid $color-white;
+            text-transform: uppercase;
+            font-size: 1em;
+            font-weight: $font-semibold;
+            letter-spacing: 0.075em;
+        }
```

Comment:
Add `text-align: center` for even spacing when there is no warning icon
<img width="254" alt="download-button" src="https://user-images.githubusercontent.com/7108785/31914451-b029ee46-b818-11e7-938e-b148f78ab6ea.png">